zephyr/rz/rz_cfg/fsp_cfg/bsp/rzg3s/bsp_cfg.h

```diff
@@ -0,0 +1,33 @@
+/*
+* Copyright (c) 2020 - 2024 Renesas Electronics Corporation and/or its affiliates
+*
+* SPDX-License-Identifier: BSD-3-Clause
+*/
+
+#ifndef BSP_CFG_H_
+#define BSP_CFG_H_
+#include "bsp_clock_cfg.h"
+#include "bsp_mcu_family_cfg.h"
+#define FSP_NOT_DEFINED 0
+#ifndef BSP_CFG_RTOS
+#if (FSP_NOT_DEFINED) != (FSP_NOT_DEFINED)
+              #define BSP_CFG_RTOS (2)
+             #elif (FSP_NOT_DEFINED) != (FSP_NOT_DEFINED)
+              #define BSP_CFG_RTOS (1)
+             #else
+#define BSP_CFG_RTOS (0)
+#endif
```

Comment:
@binhnguyen2434 : Please recheck coding style. Is it necessary for this checking while you just need "#define BSP_CFG_RTOS (0)"?

Reply:
@thaoluonguw, I have formatted the code, removed the condition and kept only "#define BSP_CFG_RTOS (0)"